internal/services/sentinel/sentinel_alert_rule_fusion_resource.go

```diff
@@ -158,25 +162,69 @@ func resourceSentinelAlertRuleFusionCreateUpdate(d *pluginsdk.ResourceData, meta
 		},
 	}
 
-	if !d.IsNewResource() {
-		resp, err := client.Get(ctx, id)
-		if err != nil {
-			return fmt.Errorf("retrieving %q: %+v", id, err)
+	if _, err := client.CreateOrUpdate(ctx, id, params); err != nil {
+		return fmt.Errorf("creating Sentinel Alert Rule Fusion %q: %+v", id, err)
+	}
```

Comment:
The string formatter for the IDs contain detailed information about the resource's name, we would be unnecessarily repeating ourselves by adding that in the error message.
```suggestion
	if _, err := client.CreateOrUpdate(ctx, id, params); err != nil {
		return fmt.Errorf("creating %s: %+v", id, err)
	}
```